Modules/xASL_module_Import.m

```diff
@@ -1,36 +1,44 @@
-function [x] = xASL_module_Import(studyPath, imParPath, studyParPath, bRunSubmodules, bCopySingleDicoms, bUseDCMTK, bCheckPermissions, x)
+function [result, x] = xASL_module_Import(x)
 %xASL_module_Import Imports the DICOM or PAR/REC source data to NIFTIs in ASL-BIDS format
 %
-% FORMAT: xASL_module_Import(studyPath[, imParPath, studyParPath, bRunSubmodules, bCopySingleDicoms, bUseDCMTK, bCheckPermissions, x])
+% FORMAT: x = xASL_module_Import(x)
```

Comment:
Output above is `[result, x]`, here only `[x]`

Reply:
387e8e19